<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuseumList</class>
 <widget class="QDialog" name="MuseumList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next Condensed</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Museum List&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mListButton">
      <property name="text">
       <string>Museum list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="itemsViewedButton">
      <property name="text">
       <string>Items Viewed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myCommentsButton">
      <property name="text">
       <string>My Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountButton">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logoMM">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>791</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="mName1_4">
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Museum 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="description1_4">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="image1_10">
      <property name="text">
       <string>image 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="image2_10">
      <property name="text">
       <string>image 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="image3_10">
      <property name="text">
       <string>image 3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="mName2_4">
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Museum 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="description2_4">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="image1_11">
      <property name="text">
       <string>Image 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="image2_11">
      <property name="text">
       <string>Image 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="image3_11">
      <property name="text">
       <string>Image 3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="mName3_4">
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Museum 3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="description3_4">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="image1_12">
      <property name="text">
       <string>Image 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="image2_12">
      <property name="text">
       <string>Image 2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="image3_12">
      <property name="text">
       <string>Image 3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="visit1_4">
      <property name="text">
       <string>Visit Museum1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="visit2_4">
      <property name="text">
       <string>Visit Museum 2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="visit3_4">
      <property name="text">
       <string>Visit Museum 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
